scripts/package/package-debian.sh

```diff
@@ -171,6 +171,9 @@ copy_files_to_debian_layout(){
 
     # Copy Manpages
     cp -a "$MANPAGE_DIR/sdk/." "$PACKAGE_LAYOUT_DIR/docs"
+
+    # Copy postinstall
+    cp "$PACKAGING_ROOT/postinst" "$PACKAGE_LAYOUT_DIR/$"
```

Comment:
I'll try again now that I know what I'm doing better. I initially was trying to put it into a folder like that but it wasn't showing up when I inspected the package. Will investigate.